### PR TITLE
new resource: moderation pocket guide for OSS repos

### PR DESCRIPTION
**Pull request by emmairwin**
a guide for teams, projects and maintainers who want to have a plan for managing their repo moderation and build healthy culture - covers topics at a lower level / more day-to-day that codes of conduct.

### CODE REVIEW

File: learning_resources/moderating-oss-repos.md [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+Here are some examples of how you might, as a team, document your approach. It's
+important to remember, that no incident will be the same, and thus some of these
+will need to be tweaked for the situation. Nonetheless it's good to have a plan!
+
+**For violations where an employee (on our team or not) is reporting or being
+repeated.**
+
+- We will follow the guidance given, and not handle this situation.
+
+**For all drive-by violations:**
```
Comment (JustinGOSSES) [inline learning_resources/moderating-oss-repos.md:267]: People might not be familiar with the term "drive-by violations" so might expand that line to `**For all drive-by violations** (meaning from someone who has never previously engaged with the project)`

File: learning_resources/moderating-oss-repos.md [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+
+At the top of the many codes of conduct you'll
+see 'Our Pledge', which then lists what's often called 'protected groups', or
+'supported groups'. These groups are those who have, historically, been
+excluded, made invisible or harmed in Open Source communities. This list signals
+accountability for the experiences of the most vulnerable and marginalized
+groups in communities, and accountability to ourselves. As such, it's important
+to be on the lookout for behaviors that may target these groups.
+
+You don't need to specifically document those which seem important to highlight
```
Comment (JustinGOSSES) [inline learning_resources/moderating-oss-repos.md:174]: Not 100% certain what this line wants to get across? Maybe could be reworded to increase clarity or explicitness.

File: learning_resources/moderating-oss-repos.md [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+### Examples
+
+Here are some examples of how you might, as a team, document your approach. It's
+important to remember, that no incident will be the same, and thus some of these
+will need to be tweaked for the situation. Nonetheless it's good to have a plan!
+
+**For violations where an employee (on our team or not) is reporting or being
+repeated.**
+
+- We will follow the guidance given, and not handle this situation.
```
Comment (JustinGOSSES) [inline learning_resources/moderating-oss-repos.md:265]: This could be read as "ignore the situation if its an employee" so maybe edit the line to be more explicit and say `We will follow the guidance given by our organization's HR department, and not attempt to handle the situation ourselves.`

File: learning_resources/moderating-oss-repos.md [hunk 4]
```diff
@@ -0,0 +1,342 @@
...
+
+- Immediately email [escalation pathway]() and request their ban on GitHub.
+- Take any other steps to ban the individual from tools immediately.
+
+**For violations described in the Code of Conduct, or explicitly in our project
+documentation by a known community member as disruptive we will..**
+
+- Give one warning then ban if they continue.
+
+### For violations where the behavior is clearly seeking to be disruptive and the person isn't known in the community
```
Comment (JustinGOSSES) [inline learning_resources/moderating-oss-repos.md:279]: Should this be `**For violations where the behavior is clearly seeking to be disruptive and the person isn't known in the community**` instead of using a header? Otherwise, it reads like the situations below are all subsets of intentional distruption.

**Review by JustinGOSSES — CHANGES_REQUESTED** [review]

File: learning_resources/moderating-oss-repos.md (hunk 3 above)
Comment (emmairwin) [reply learning_resources/moderating-oss-repos.md:265]: I just removed this part
